app/windows/Storage/fileIntegration.js

```diff
@@ -114,20 +118,31 @@ export function setupAddAppOnDrop () {
  * Prompt the user if he is sure that we should remove a file
  * return a Promise
  */
-export function proptAndRemoveObjects (hashes) {
-  const buttons = ['Abort', 'Of course, Duh!']
+export function proptAndRemoveObjects (elements) {
+  const isOneFile = elements.length === 1
+
+  const title = isOneFile ? 'Delete file' : 'Delete files'
+  const message = `Are you sure you want to delete the selected ${isOneFile ? 'file?' : `${elements.length} files?`}`
+  const elementsDetails = elements
+    .map(el => {
+      return `${el.hash.substring(0, 25)} - ${el.stat.CumulativeSize.value} ${el.stat.CumulativeSize.unit}`
+    })
+    .join('\n')
+
   const opts = {
-    title: 'Continue?',
-    message: `Are you sure you want to delete ${hashes.length} files?`,
-    detail: `This includes: \n${hashes.join('\n')}`,
-    buttons,
-    cancelId: 0
+    title,
+    message,
+    detail: `This includes: \n\n${elementsDetails}\n\nYou can't undo this action.`,
```

Comment:
What about `Once done, You can't restore the files` instead of `You can't undo this action.` (as if it already happened?)
What do you say?